pkg/module/go_get_fetcher.go

```diff
@@ -146,5 +146,5 @@ func getRepoDirName(repoURI, version string) string {
 
 // getPackagePath returns the path to the module cache given the gopath and module name
 func getPackagePath(gopath, module string) string {
-	return filepath.Join(gopath, "src", "mod", "cache", "download", module, "@v")
+	return filepath.Join(gopath, "pkg", "mod", "cache", "download", module, "@v")
```

Comment:
this as well

https://github.com/gomods/athens/blob/441e21d28113ad3de1e1fd757300ee4d9cd7804b/pkg/module/go_get_fetcher.go#L38

Reply:
sorry, ignore,